src/components/atoms/M/M.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import { DEFAULT_MESSAGES } from "assets/messages";
+
+const findMessage: (key: string) => string = (key) => DEFAULT_MESSAGES[key];
+
+export const M: React.FC = ({ children }) => {
```

Comment:
Short name for a component that might be seen in plenty of places. Just like `for( i )` strives not to waste space for common concept.